CHANGELOG.md

```diff
@@ -12,6 +12,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Fixed
 
 - Check if the user specified for an exchange backup operation has a mailbox.
+
+### Changed
+
+- msgraph-beta-sdk-go replaces msgraph-sdk-go for new features. This can lead to long build times. 
```

Comment:
Is this still true if we're using a minimized/sliced version of the beta package?

Reply:
This is not true. I have recompiled the code on several occasions. As of now, we are not experiencing the build time errors that were presented previously. The main reason is all the abstract kiota libraries are imported with msgraph. The build takes approx. 90 minutes with a cache miss. This is similar to our current scenario.

Reply:
Resovled within [cd51aae](https://github.com/alcionai/corso/pull/2205/commits/cd51aaea7c693a930cf996604f7b537926b6d2d0)